packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -15,8 +12,8 @@ class GoogleMapController {
   GoogleMapController._(
     CameraPosition initialCameraPosition,
     this._googleMapState, {
-    @required this.mapId,
-  }) : assert(_googleMapsFlutterPlatform != null) {
+    required this.mapId,
+  }) : assert(GoogleMapsFlutterPlatform.instance != null) {
```

Comment:
if `instance` is non-nullable, this assertion is unnecessary.

Reply:
Argh, I really don't understand why that's not an error post-migration.

Reply:
yeah, i have seen analyzer not catching errors like this too.